src/style.css

```diff
@@ -0,0 +1,136 @@
+@font-face {
+  font-family: Avenir_heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: avenir_book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+}
```

Comment:
<img width="850" alt="Screenshot 2023-07-23 at 16 05 36" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/135232001/c7bbbce6-98f1-4357-8e1a-ae8a3feee657">
Consider fixing comments from the previous review